src/unicef_attachments/fields.py

```diff
@@ -47,17 +48,16 @@ def get_attachment(self, instance):
         return None
 
     def get_attribute(self, instance):
-        attachment = self.get_attachment(instance)
-        return attachment.file if attachment is not None else None
+        return self.get_attachment(instance)
```

Comment:
will this have any consequences on backwards compatibility?

Reply:
rest this logic was moved to PermittedAttachmentField.get_attribute